example_tracking.js

```diff
@@ -8,7 +8,7 @@ a tracking number as well as how to register a webhook for tracking updates
 var shippo = require('shippo')('<YOUR_PRIVATE_KEY>');
 
 // example Tracking object for tracking a shipment
-shippo.track.get_status('usps', '1122334455667788')
+shippo.track.get_status('shippo', 'SHIPPO_TRANSIT')
```

Comment:
what's the rationale behind changing the example? I think there's some value in having example values that resemble real values